app/components/embed/file/file_row_component.html.erb

```diff
@@ -17,7 +17,7 @@
   <td role="gridcell">
     <% if download? %>
       <a href="<%= url %>" target='_blank' rel='noopener noreferrer' download>
-        <i class='sul-i-download-3'></i> Download
+        <i class='mui-download-icon'></i> Download
```

Comment:
`<i>` seems like the wrong semantic element to use for an icon.  https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i  Can we just inline the svg?

Reply:
Yeah. Updated PR